### PR TITLE
[ALLUXIO-2119] Reduce use of whitebox in AbstractFileSystemTest

### PR DESCRIPTION
**Pull request by aaudiber**
https://alluxio.atlassian.net/browse/ALLUXIO-2119

### CODE REVIEW

File: core/client/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java [hunk 1]
```diff
@@ -123,48 +121,36 @@ private URL getSourcePath(Class<?> clazz) {
...
     Configuration.set(Constants.MASTER_RPC_PORT, Integer.toString(uri.getPort()));
     Configuration.set(Constants.ZOOKEEPER_ENABLED, "true");
 
     final org.apache.hadoop.fs.FileSystem fs = org.apache.hadoop.fs.FileSystem.get(uri, conf);
-
     Assert.assertTrue(fs instanceof FaultTolerantFileSystem);
-
-    PowerMockito.verifyStatic();
-    alluxio.client.file.FileSystem.Factory.get();
-    ClientTestUtils.resetClient();
```
Comment (calvinjia) [inline core/client/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java]: No longer necessary to reset the client?

Comment (aaudiber) [inline core/client/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java]: it's already being reset in `@After`

**Comment by aaudiber** [comment]
@calvinjia Comments addressed, PTAL

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**